docs: improve doc (#1568)
NVIDIA/NeMo-Guardrails: docs/configure-rails/overview.md

# NeMo Guardrails Library Configuration Overview

> Before using the NeMo Guardrails library, you need to prepare configuration files that define your guardrails behavior. When you initialize the library's core classes or the `nemoguardrails` CLI chat or server, it will load the configuration files you'll create in the next chapter [](../run-rails/index.md). This section provides complete instructions on preparing your configuration files and executable scripts.

Reviewer: ```suggestion
Before using the NeMo Guardrails library, you need to prepare configuration files that define your guardrails behavior. When you initialize the library's core classes or the `nemoguardrails` CLI chat or server, it will load these configuration files as shown in the next chapter [](../run-rails/index.md). This section provides complete instructions on preparing your configuration files and executable scripts.
```
